roles/puppet_repositories/tasks/debian.yml

```
@@ -5,6 +5,12 @@
     state: present
     update_cache: true
 
+- name: Remove any existing Puppet release packages
```

```suggestion
- name: Remove any existing Puppet release packages for versions other than {{ puppet_repositories_version }}
```

Otherwise users will freak out XD